CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## [next] 
 - Adding Component#onDestroy
 - Adding Keyboard events API
+- Adding Box2DGame, an extension of BaseGame to simplify lifecycle of Box2D components
```

Comment:
If you rebase your changes instead next time the merge conflicts will be a lot easier to solve, and it will keep the commit log cleaner. :)

Reply:
Thanks @spydon next time will do that ;)